rct229/rulesets/ashrae9012019/section19/section19rule10.py

```diff
@@ -169,53 +170,61 @@ def get_calc_vals(self, context, data=None):
                         baseline_system_types_b = sys_type
 
                 fan_sys_b = hvac_b["fan_system"]
+                fan_sys_p = hvac_p["fan_system"]
                 fan_air_economizer_b = find_one("$.air_economizer", fan_sys_b)
+                fan_air_economizer_p = find_one("$.air_economizer", fan_sys_p)
                 fan_air_economizer_type_b = find_one("$.air_economizer.type", fan_sys_b)
+                fan_air_economizer_type_p = find_one("$.air_economizer.type", fan_sys_p)
 
                 return {
                     "hvac_id_b": hvac_id_b,
                     "baseline_system_types_b": baseline_system_types_b,
                     "fan_air_economizer_b": fan_air_economizer_b,
+                    "fan_air_economizer_p": fan_air_economizer_p,
                     "fan_air_economizer_type_b": fan_air_economizer_type_b,
+                    "fan_air_economizer_type_p": fan_air_economizer_type_p,
                     "HVAC_systems_primarily_serving_comp_rooms_list": HVAC_systems_primarily_serving_comp_rooms_list,
```

Comment:
Is the reason to add proposed data because we do not have is_economizer_modeled_in_proposed function?

Reply:
This was my proposed solution to reach this case of the RDS https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section19/Rule19-10.md?plain=1#L55

I think `is_economizer_modeled_in_proposed` could probably be used too